suite-native/app/src/App.tsx

```diff
@@ -58,12 +62,25 @@ const AppComponent = () => {
             } catch (error) {
                 Alert.alert('Error', error?.message ?? 'Unknown error');
                 console.error(error.message);
+            } finally {
+                // Tell the application to render
+                setIsAppReady(true);
             }
         };
         initActions();
     }, [dispatch, isConnectInitialized]);
 
-    if (!isConnectInitialized) {
+    useEffect(() => {
+        const checkSplashScreenHide = async () => {
```

Comment:
I think you don't need async/await here because you don't care about result anyway so just:

```tsx
if (isAppReady) {
  SplashScreen.hideAsync();
}
```
should work same

Reply:
6eb52e06277547303899f873a87bcd4f9fcddf50